data/src/main/java/com/globant/data/mapper/BaseMapper.kt

```diff
@@ -1,5 +1,8 @@
 package com.globant.data.mapper
 
-interface BaseMapper<E, D> {
-    fun transform(type: E, resourcesMap: MutableMap<String, Pair<Int, Int>>? = null): D
+interface BaseMapper<E, D, R> {
+    fun transform(type: E, resources: R? = null): D
+    // This BaseMapper needed a third generic value to handle the different hash maps that
+    // deliver the resources for the entities, and it's nullable and initialized null to avoid
+    // errors on parameters insertions
```

Comment:
i would remove this comment. But if you want to document its funtionallity, I would use the  /** **/ coment on the top of the function

Reply:
done, thanks for the advice